app/src/pages/clinic.tsx

```diff
@@ -184,12 +185,12 @@ const Clinic: NextPage<Props> = (props: Props) => {
                           className="usa-radio__label"
                           htmlFor={clinic.clinic}
                         >
-                          {clinic.clinic}
-                          <span className="usa-checkbox__label-description">
-                            {clinic.clinicAddress}
-                            <br />
-                            {clinic.clinicTelephone}
-                          </span>
+                          <ClinicInfo
+                            name={clinic.clinic}
+                            streetAddress={clinic.clinicAddress}
+                            phone={clinic.clinicTelephone}
+                            isFormElement={true}
```

Comment:
Hmmm it looks like in general react recommends explicitly passing a `{true}` value to avoid confusion: https://reactjs.org/docs/jsx-in-depth.html#props-default-to-true